components/Chips/examples/ChipsActionExampleViewController.m

```diff
@@ -33,6 +34,13 @@ - (id)init {
   return self;
 }
 
+- (MDCContainerScheme *)containerScheme {
+  MDCContainerScheme *scheme = [[MDCContainerScheme alloc] init];
```

Comment:
Would you please add the typography scheme either here or as a follow-up PR?

Reply:
done